community/kernel/src/main/java/org/neo4j/procedure/Procedure.java

```diff
@@ -87,6 +90,7 @@
  * <ul>
  *     <li>{@link org.neo4j.graphdb.GraphDatabaseService}</li>
  *     <li>{@link org.neo4j.logging.Log}</li>
+ *     <li>{@link org.neo4j.kernel.api.KernelTransaction}</li>
```

Comment:
`KernelTransaction` is not public API, so we don't want to document that this is possible.